test/unit/lib/format-time.test.js

```diff
@@ -0,0 +1,87 @@
+const format = require('../../../src/lib/format-time');
+
+describe('unit test lib/format-time.js', () => {
+    let realDateNow;
+    let _mockFormat;
+    const mockFormatExpression = {
+        format: jest.fn()
+    };
+    beforeEach(() =>{
+        realDateNow = Date.now.bind(global.Date);
+        global.Date.now = () => 0;
+        _mockFormat = Intl.RelativeTimeFormat = jest
+            .fn()
+            .mockImplementation(() => mockFormatExpression);
+
+    });
+    afterEach(()=>{
+        global.Date.now = realDateNow;
+        jest.resetAllMocks();
+    });
+
+    test('test timestamp that is 2 minutes in the future', () => {
+        let response;
+        const twoMin = 2 * 60 * 1000;
+        mockFormatExpression.format.mockReturnValue('in 2 minutes');
+        response = format.formatRelativeTime(twoMin, 'en');
+        expect(mockFormatExpression.format).toHaveBeenCalledWith(2, 'minute');
+        expect(response).toEqual('in 2 minutes');
+    });
+
+    test('test rounding timestamp that is 4.4 minutes rounds to 4', () => {
+        let response;
+        const twoMin = 4.4 * 60 * 1000;
```

Comment:
```suggestion
        const fourMin = 4.4 * 60 * 1000;
```

Reply:
Thanks for the review! Made all of the changes. I'll commit once I confirm travis tests pass.